src/audio/component.c

```diff
@@ -505,3 +505,51 @@ int comp_copy(struct comp_dev *dev)
 
 	return ret;
 }
+
+#if CONFIG_IPC_MAJOR_4
+static uint32_t get_sample_group_size_in_bytes(const struct ipc4_audio_format fmt)
+{
+	return (fmt.depth >> 3) * fmt.channels_count;
+}
+
+static uint32_t get_one_ms_in_bytes(const struct ipc4_audio_format fmt)
+{
+#ifdef ROUND_UP
```

Comment:
in which cases is `ROUND_UP` not defined?

Reply:
There are some builds that don't recognize this. It is defined in Zephyr so I 'm guessing those builds don't use Zephyr (not sure which one it was). This makes it awkward, as I'd rather just use the macro , but it seems I need to expand it anyway because builds will fail otherwise.